particles/Node.hpp

```diff
@@ -13,7 +13,7 @@ class Node : public score::gfx::NodeModel
   score::gfx::NodeRenderer*
   createRenderer(score::gfx::RenderList& r) const noexcept override;
   void process(const score::gfx::Message& msg) override;
-  float font;
+  float particlesSpeedMod;
```

Comment:
one last thing : initialize this to something (e.g. 0.f). 
This is because otherwise it's uninitialized and the bits will be entirely random. And unlike ints, which "just" have a random value (which can already be fairly damaging or make bugs hard to understand), some CPUs have what is called "trap representation" for some floats, which means that merely accessing a float with an invalid bit pattern will cause a CPU exception (and immediately crash the program).